apps/src/code-studio/pd/foorm/editor/components/FoormEntityEditor.jsx

```diff
@@ -117,6 +118,14 @@ class FoormEntityEditor extends React.Component {
       <div style={styles.options}>
         <div>
           <form>
+            <label>
+              workshop_id <br />
+              <input
+                type="text"
+                value={this.state.workshop_id}
+                onChange={e => this.setState({workshop_id: e.target.value})}
+              />
+            </label>
```

Comment:
shouldn't this be a hidden input? my understanding is that the teacher goes to the form, and the value of the workshop id is determined by their enrollment code. I don't see why we would want them to be able to change it, but it's possible I am missing something. if we can make it hidden, that seems worth doing to make sure users can't accidentally mess up our data.

Reply:
This is the Preview area in the Foorm Editor that I showed above that allows you to test different variables on your survey easily